api/users/serializers.py

```diff
@@ -56,9 +60,15 @@ class Meta:
 class UserListSerializer(serializers.ModelSerializer):
     role = serializers.SerializerMethodField(read_only=True)
     join_date = serializers.SerializerMethodField(read_only=True)
+    if settings.IS_RBAC_INSTALLED:
+        roles = UserRoleSerializer(many=True, read_only=True, source="get_user_roles")
```

Comment:
I'd love to see some tests for this. It might be tough to do here though because of the dependency on the rbac library. I wonder if this would be better suited as a new endpoint, which lives in the RBAC repository?

Reply:
I saw that you had a similar comment in another PR, yes I think you are right, the serializers would no longer depend on RBAC and could be tested more easily

Reply:
The endpoint was created in RBAC: https://github.com/Flagsmith/flagsmith-rbac/pull/15